config/production.json

```diff
@@ -0,0 +1,6 @@
+{
+  "port": 5000,
```

Comment:
What is this file used for?

Reply:
It is used for setting the redis and mongo connections. They differ in development and production

Reply:
But the environment should not matter at all since those values should be provided by ENV variables?

Reply:
It should be with the rest of the stuff configured with convict.